nox/_options.py

```diff
@@ -23,6 +23,13 @@
 from nox import _option_set
 from nox.tasks import discover_manifest, filter_manifest, load_nox_module
 
+if sys.version_info < (3, 8):  # pragma: no cover
+    from typing_extensions import Literal
+else:  # pragma: no cover
+    from typing import Literal
```

Comment:
```suggestion
if sys.version_info < (3, 8):
    from typing_extensions import Literal
else:
    from typing import Literal
```

I think you can remove the `pragma` comments here. Coverage for the two is stored in a separate file (`.coverage.3.x`) and combined by the `cover` session.

Reply:
Good point, I decided to keep it the same as the rest of the code (e.g. [command.py](https://github.com/wntrblm/nox/blob/main/nox/command.py#L27)) for consistency.